base/src/MotionVectorExtractor.cpp

```diff
@@ -290,8 +318,19 @@ void DetailOpenH264::getMotionVectors(frame_container& frames, frame_sp& outFram
 			frames.insert(make_pair(motionVectorPinId, outFrame));
 		}
 	}
-
-	if ((!sDecParam.bParseOnly) && (pDstInfo.pDst[0] != nullptr) && (mMotionVectorSize != mWidth * mHeight * 8))
+	if(sendOverlayFrame == false){
```

Comment:
what is this change about ? why is this reqd in the sample PR

Reply:
This is required to determine whether the frames are in the motion or not when we set the sendOverlayFrame to false.